actors/builtin/miner/miner_actor.go

```diff
@@ -1619,6 +1606,26 @@ func getVerifyInfo(rt Runtime, params *SealVerifyStuff) *abi.SealVerifyInfo {
 	}
 }
 
+// Closes down this miner by erasing its power, terminating all its deals and burning its funds
+func terminateMiner(rt Runtime) {
+	var st State
+	rt.State().Readonly(&st)
+
+	// Notify power actor with lock-up total being removed.
+	_, code := rt.Send(
```

Comment:
It would be nice to extract miner termination into a function like this, but only in so much as we use it for other (yet to be impled/specced out) cases.  Including the consensus fault specific message send inside `terminateMiner` seems to defeat the purpose since we can't use this in other code paths.

I would either move the send up and leave only the two other calls, or defer wrapping miner termination into its own function at all.

Reply:
Good catch.

Reply:
I removed the call to storage market, but left the method so at least deal termination will be tied to miner termination.